eureka-client/src/main/java/com/netflix/discovery/DiscoveryClient.java

```diff
@@ -768,7 +768,7 @@ public InstanceInfo getNextServerFromEureka(String virtualHostname, boolean secu
                     + virtualHostname);
         }
         Applications apps = this.localRegionApps.get();
-        int index = (int) (apps.getNextIndex(virtualHostname.toUpperCase(Locale.ROOT),
+        int index = (int) (apps.getNextIndex(virtualHostname,
```

Comment:
yes, it's been moved into Applications.getNextIndex for consistency (previously Applications class applies 'toUpperCase' to some of its other method parameters but not this one)